packages/shared/styles/components/organisms/SfHeader.scss

```diff
@@ -1,37 +1,30 @@
 @import "../../helpers";
 .sf-header-navigation-item {
-  flex: 1;
+  position: var(--header-navigation-item-position, relative);
   display: flex;
-  & > *:not(.sf-mega-menu) {
-    --link-color: var(--header-navigation-item-color, var(--c-dark-variant));
-    box-sizing: border-box;
-    flex: 1;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    padding: var(--header-navigation-item-padding, calc(var(--spacer-sm) - 1px) var(--spacer-sm));
-    margin: var(--header-navigation-item-margin, 0);
-    @include border(--header-navigation-item-border, 0 0 2px 0, solid, transparent);
-    transition: var(--header-navigation-item-transition, border-color 150ms ease-in-out, color 150ms ease-in-out);
-    color: var(--header-navigation-item-color, var(--c-dark-variant));
-    @include font(
-      --header-navigation-item-font,
-      var(--font-bold),
-      var(--font-sm),
-      1.2,
-      var(--font-family-secondary)
-    );
-    text-transform: uppercase;
-    text-decoration: initial;
-    text-align: center;
-    @include for-desktop {
-      padding: var(--header-navigation-item-padding, calc(var(--spacer-lg) - 1px) var(--spacer-sm));
-      margin: var(--header-navigation-item-margin, 0);
-    }
-    &:focus {
-      --header-navigation-item-color: var(--c-text);
-      --header-navigation-item-border-color: var(--c-primary);
-    }
+  flex: var(--header-navigation-item-flex, 0);
+  @include for-desktop {
+    --header-navigation-item-flex: 1;
+    --header-navigation-item-position: static
+  }
+  &__content {
+    position: var(--header-navigation-item-content-position, fixed);
+    top: var(--header-navigation-item-content-position, 0);
+    width: var(--header-navigation-item-content-width, 100%);
+    z-index: var(--header-navigation-item-content-position, 2);
+    --header-navigation-item-padding: 0;
+  }
+  &__link {
+    --link-text-decoration: var(--header-navigation-item-text-decoration, none)
+  }
+  & > &__menu-item{
+    --menu-item-font-size: var(--font-base);
+    --menu-item-font-weight: var(--font-medium);
+    --menu-item-text-transform: uppercase;
+    display: var(--header-navigation-item-menu-item-display, flex);
```

Comment:
Is it necessary to add display: flex, because sf-menu-item  has this property originally set, wdyt?